config/helm/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 resources:
 - ../crd
 commonAnnotations:
-  "helm.sh/resource-policy": keep
```

Comment:
Why this was removed ? we can't remove this please keep it

Reply:
this should be fixed back to the way it was now.